.travis.yml

```
@@ -0,0 +1,26 @@
+language: python
+python:
+  - 3.6
+cache:
+  directories:
+    - $HOME/.cache/pip
+    - $HOME/.ccache  # https://github.com/travis-ci/travis-ci/issues/5853
+
+install:
```

I think the packages for Travis should be added as well: `coverage`, `codecov`, `flake8`.
```suggestion
install:
  - pip install coverage codecov flake8
```